tests/chainer_tests/testing_tests/test_math_function_test.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from chainer import functions as F
+from chainer import testing
+
+
+class TestNoNumpyFunction(unittest.TestCase):
+
+    def test_no_numpy_function(self):
+        with self.assertRaises(ValueError):
+            testing.math_function_test(F.rsqrt)  # no numpy.rsqrt
+
+
+class TestInvalidExpectedLabel(unittest.TestCase):
+
+    def test_invalid_expected_label(self):
+        with self.assertRaises(ValueError):
+            # no numpy.rsqrt
```

Comment:
Yes for testing invalid expected label with actual label, but what I intended here was giving `label_expected` for testing a function that does not have corresponding `chainer.Function` subclass, so the comment was less appropriate. I fixed it.

Relating the discussion below.
> According to the code that uses this decorator here, it must be a python function that has corresponding Chainer function, rather than the Chainer function itself.
> Alternatively, directory pass Chainer Functions to this decorator as func argument and do not use func_class (I prefer it to the above change).